algobot/helpers.py

```diff
@@ -40,6 +44,49 @@
 LONG_INTERVAL_MAP = {v: k for k, v in SHORT_INTERVAL_MAP.items()}
 
 
+class Paths:
+
+    def __init__(self, root_dir: str, app_dirs):
```

Comment:
Can we add a hint for app_dirs?

Reply:
Yes - was thinking about how we could inject a mock version for testing but it does make sense.

Reply:
Added `AppDirTemp` which is used when `ALGOBOT_TESTING` env var is set.

Not 100% happy with this approach as test code resides in app, but not sure of a cleaner way.

This is set in `conftest.py` which is executed before the test suite, and teared down on shutdown

Reply:
Done